jujugui/static/gui/src/app/components/deployment-bar-deploy-button/_deployment-bar-deploy-button.scss

```diff
@@ -0,0 +1,12 @@
+.deployment-bar__deploy-button {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+
+    &--active {
+        @extend %button-orange;
+    }
+
+    &:not(&.deployment-bar__deploy-button--active) {
+            @extend %button-disabled;
+    }
```

Comment:
Same here, I think we would be better to have one class styling an element instead of two.